language/bytecode-verifier/src/verifier.rs

```diff
@@ -21,9 +21,11 @@ use vm::{
 /// that do not require full control over verification
 pub fn verify_module(module: &CompiledModule) -> VMResult<()> {
     DuplicationChecker::verify_module(&module)?;
+    unused::verify_module(&module)?;
```

Comment:
You need to add these checks to the loader as well

Reply:
I think what I did at the loader is to replace each individual `verify_module` and replace it with an umbrella `bytecode_verifier::verify_module()` there.

Reply:
You'll need to cleanup that comment above this then :P 

I'm totally fine with this though, and I don't know why Dario was always so paranoid about sharing this